### ISSUE TITLE
page.MustSearch().MustClick() Error

### ISSUE THREAD

**Issue by jaekook-neonesia** [go-rod/rod#249]
**Rod Version:** v0.73.1

Hi, I have a question for element click inside frame.
Question is, 
Can I use MustSearch for click, input for inside frame element.
Or use MustElement().MustFrame() instead MustSearch.

I thought MustSearch also support MustClick().
Thank you.

When I try to use MustSearch("inside frame element").MustClick() then I got below error.

Code : page.MustSearch(si.TopMenu).MustClick()

```
panic: {-32000 Argument should belong to the same JavaScript world as target object }

goroutine 66 [running]:
github.com/go-rod/rod/lib/utils.E(0xc00020d4c8, 0x1, 0x1, 0x0, 0x0, 0x0)
        /go/pkg/mod/github.com/go-rod/rod@v0.73.1/lib/utils/utils.go:51 +0x15d
github.com/go-rod/rod.(*Element).MustClick(0xc00069c150, 0x0)
        /go/pkg/mod/github.com/go-rod/rod@v0.73.1/must.go:653 +0x125

```

So, I switch to use MustElement().MustFrame() like below.
```
f := page.MustElement(si.TopFrame).MustFrame()
f.MustElement(si.TopMenu).MustClick()

```
This code worked without error.

**Comment by ysmood** [comment]
Yes, as long as it's an `rod.Element`, it should support all operations on it, the unit test works fine to me:

https://github.com/go-rod/rod/blob/04b99d1f9958bf14599dd9a2c7d7937915489897/query_test.go#L106-L120

Maybe you should check the latest version, recently I fixed some bugs.

**Comment by ysmood** [comment]
Can you provide minimal code for us to reproduce your issue?

**Comment by jaekook-neonesia** [comment]
Code
si.TopFrame = "html > frameset > frame"
si.TopMenu = "#tableheader > tbody > tr > td.right-main-header > div.menu-header > div:nth-child(1) > div:nth-child(3) > a"
------------- Worked
	f := page.MustElement(si.TopFrame).MustFrame()
	f.MustElement(si.TopMenu).MustClick()

------------- Error
        page.MustSearch(si.TopMenu).MustClick()

------------- Also Error
	elm := page.MustSearch(si.TopMenu)
	if elm == nil {
		err := fmt.Errorf("Cant Search : %s", si.TopMenu)
		log.Error(err)
		return err
	}

	elm.MustClick()                                         <----------- Error here



-------------- Page source
<frameset rows="107px,*" framespacing="0" border="0" frameborder="NO">
	    <frame src="./Management System_files/Header.html" name="head" noresize="" scrolling="NO">
	    <frameset cols="218px,*" framespacing="0" border="0" frameborder="NO">
    	        <frame src="./Management System_files/LeftMenu.html" border="1" name="menu" noresize="">
							<frame src="./Management System_files/Welcome.html" name="channel">
</frameset>
</frameset>

-------------- Top frame source
<div class="menu-header"><div style="float:left"><div class="menu-item"><a class="menu-item" onclick="loadtwo(&#39;LeftMenu.aspx&#39;,&#39;Welcome.aspx&#39;) " href="https://-----/Header.aspx#top">Home</a></div><div class="menu-item-padding">|</div><div class="menu-item"><a class="menu-item" href="https://-----/LeftMenu.aspx?id=4" target="menu">Account Information</a></div><div class="menu-item-padding">|</div><div class="menu-item"><a class="menu-item" href="https://-----/LeftMenu.aspx?id=5" target="menu">Fund Transfer</a></div><div class="menu-item-padding">|</div></div>
    <div style="float:right;padding-right:10px"><form name="header" method="post" action="https://-----/Header.aspx" id="header">

**Comment by ysmood** [comment]
Thank you for your reply. Can you provide a more specific sample? Like this file: https://github.com/go-rod/rod/blob/master/lib/examples/disable-window-alert/main.go

So that I can just run it and reproduce the issue directly.

**Comment by jaekook-neonesia** [comment]
I will check it.
My guess is  Element->page is different.

`	elmT := page.MustSearch(si.TopMenu)
	ft := page.MustElement(si.TopFrame).MustFrame()
	elmF := ft.MustElement(si.TopMenu)

	log.Debug("Search Session:", elmT)
	log.Debug("Frame Session:", elmF)
`
`
2020-10-21T04:44:20Z debug layer=debugger nexting
{"level":"debug","msg":"Search Session:&{0xc00002a078 0xb3b260 0xc000360120 0xc0001ac180}"}
2020-10-21T04:44:21Z debug layer=debugger nexting
{"level":"debug","msg":"Frame Session:&{0xc00002a078 0xb3b260 0xc000146090 0xc0001ac600}"}
`

**Comment by ysmood** [comment]
@jaekook-neonesia I just released v0.74.1 which has some bug fix for iframe, please check it.
